src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,35 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int operatingWeight;
```

Comment:
The field 'operatingWeight' should have an access modifier to encapsulate the field properly. Consider using private access if it should not be accessible outside this class or protected if it should be accessible in subclasses as well.